packages/embed-react/src/components/make-button-component.tsx

```diff
@@ -6,21 +6,22 @@ import React, {
   useMemo,
   useRef,
   ReactHTML,
-  HTMLAttributes,
+  AriaAttributes,
 } from 'react'
+import { ButtonProps } from '@typeform/embed'
 
 import { InlineStyle } from './inline-style'
 
 type ButtonComponentBaseProps = {
   id: string
   as?: keyof ReactHTML
-  buttonProps?: HTMLAttributes<HTMLElement> & Record<string, string>
+  buttonProps?: ButtonProps & AriaAttributes
```

Comment:
This fixes the failing tests that were missing proper aria typing. 
cc: @mathio